support-frontend/assets/pages/contributions-landing/components/AmazonPay/AmazonPayCheckout.tsx

```diff
@@ -1,28 +1,27 @@
 import type { ContributionType } from 'helpers/contributions';
 import type {
 	AmazonLoginObject,
-	AmazonPayData,
 	AmazonPaymentsObject,
 } from 'helpers/forms/paymentIntegrations/amazonPay/types';
 import AmazonPayLoginButton from './AmazonPayLoginButton';
 import AmazonPayWallet from './AmazonPayWallet';
 
-interface AmazonPayProps {
+interface AmazonPayCheckoutProps {
```

Comment:
Unrelated, just noticed the props type didn't quite match the naming convention!